google-cloud-storage/lib/google/cloud/storage/policy/bindings.rb

```diff
@@ -86,27 +101,27 @@ def insert *bindings
           end
 
           ##
-          # Deletes a binding or bindings from the collection.
+          # Deletes the binding or bindings from the collection that are equal to
+          # the arguments. The specification arguments may be
+          # {Google::Cloud::Storage::Policy::Binding} objects or equivalent hash
+          # objects that will be implicitly coerced to binding objects.
           #
-          # @param [Google::Cloud::Storage::Policy::Binding] bindings One or
-          #   more bindings to be removed from the policy owning the
-          #   collection.
+          # @param [Google::Cloud::Storage::Policy::Binding, Hash] bindings One
+          #   or more specifications for bindings to be removed from the
+          #   collection. The arguments may be
+          #   {Google::Cloud::Storage::Policy::Binding} objects or equivalent
+          #   hash objects that will be implicitly coerced to binding objects.
           #
           # @example
           #   require "google/cloud/storage"
           #
           #   storage = Google::Cloud::Storage.new
-          #   bucket = storage.bucket "my-todo-app"
+          #   bucket = storage.bucket "my-bucket"
           #
           #   bucket.policy requested_policy_version: 3 do |p|
           #     p.bindings.remove({
           #                         role: "roles/storage.admin",
-          #                         members: ["user:owner@example.com"],
-          #                         condition: {
-          #                           title: "test-condition",
-          #                           description: "description of condition",
-          #                           expression: "expr1"
-          #                         }
+          #                         members: ["user:owner@example.com"]
```

Comment:
How does a user delete a conditional binding?

Reply:
Same way, by spec. I can re-add the condition to the example if you think it's more helpful.

Reply:
It would be helpful to see an example, could you please re-add the example, thank you!

Reply:
Restored.